docs/UserGuide.md

```diff
@@ -758,11 +797,63 @@ Exit the application.
 
 ### Export Data
 
-<!-- TODO: fill this in -->
+> `export {file_path} [/overwrite]`
+
+Export all modules progress to a JSON-format file. 
+
+- <span style="color:#e46c0a">`file_path`</span> : The path of the file to export the modules progress to
+  - Must be relative to Le Tracker's default saving directory (:exclamation:The default saving directory is `[JAR 
+    file location]/data`)
+  - Must not coincide with Le Tracker's current tracker file path. (:exclamation:The default tracker file path is `
+    [JAR file location]/data/letracker.json`)
+  - If `/overwrite` is not specified, the file specified in `file_path` must not exist
+- <span style="color:#e46c0a">`/overwrite`</span> : If specified, Le Tracker will overwrite all data in 
+  the file specified in `file_path` with the current modules progress in Le Tracker
+  - If the file specified in `file_path` exists, it must allow reading and writing operations
+  - If the file specified in `file_path` exists, it must be of JSON format
+  - If the file specified in `file_path` exists, the flag `/overwrite` will be ignored
+
+Examples: 
+
+- `export hello.json`
+
+- `export /../../haha.json /overwrite`
 
 ### Import Data
 
-<!-- TODO: fill this in -->
+> `import {file_path} [/mod {module_1}[, {module_2}[, {module_3}, ...]]] [/overwrite]`
+
+Import modules progress from a JSON-format file to the current tracker.
+
+- <span style="color:#e46c0a">`file_path`</span> : The path of the file to export the modules progress from
+  - Must be relative to Le Tracker's default saving directory (:exclamation:The default saving directory is `[JAR
+    file location]/data`)
+  - The file specified in `file_path` must exist. (:exclamation: If only the file's name is specified, the file must 
+    exist in the default saving directory at `[JAR file location]/data`)
+  - Must allow reading and writing operations
+  - Must be of JSON format
```

Comment:
Don't think there's a need to mention this since it's already covered under "valid Le Tracker data file"